examples/07.error_handling/hello.cc

```diff
@@ -2,18 +2,20 @@
 // SPDX-License-Identifier: MIT
 
 #include "hello.h"
+#include <debug.hh>
 #include <fail-simulator-on-error.h>
 
 /// Thread entry point.
-void __cheri_compartment("hello") entry()
+int __cheri_compartment("hello") entry()
 {
 	// Try writing a string with a missing null terminator
 	char maliciousString[] = {'h', 'e', 'l', 'l', 'o'};
-	write(maliciousString);
+	(void)write(maliciousString);
```

Comment:
Should this check for `ECOMPARTMENTFAIL`?

Reply:
Done throughout